src/braket/ir/ahs/program_v1.py

```diff
@@ -0,0 +1,51 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+from pydantic import BaseModel, Field
+
+from braket.ir.ahs.atom_array import AtomArray
+from braket.ir.ahs.hamiltonian import Hamiltonian
+from braket.schema_common import BraketSchemaBase, BraketSchemaHeader
+
+
+class Setup(BaseModel):
```

Comment:
Why is this the lone one not in its own class?

Reply:
I may be not answering your question, but just for context: We introduce this extra layer (Setup) on top of the AtomArray, because later, we expect Setup to contain more object other than Atom array. Objects that describe the physical setup, independent of the Hamiltonian.